ledger/src/blockstore_processor.rs

```diff
@@ -106,6 +112,19 @@ fn execute_batch(
         transaction_status_sender.is_some(),
     );
 
+    if let Some(replay_votes_sender) = replay_votes_sender {
+        for (transaction, (processing_result, _)) in
+            OrderedIterator::new(batch.transactions(), batch.iteration_order())
```

Comment:
@t-nelson double checking, is this the right way to match the transaction results with the transactions?

Reply:
Yep! :100: